src/VisualStudio/Core/Def/DocumentOutline/Extensions/SortDescriptionCollectionExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.ComponentModel;
+
+namespace Microsoft.VisualStudio.LanguageServices.DocumentOutline
+{
+    internal static class SortDescriptionCollectionExtensions
+    {
+        private static ImmutableArray<SortDescription> NameSortDescriptions { get; } = ImmutableArray.Create(new SortDescription("Data.Name", ListSortDirection.Ascending));
+        private static ImmutableArray<SortDescription> LocationSortDescriptions { get; } = ImmutableArray.Create(new SortDescription("Data.RangeSpan.Start.Position", ListSortDirection.Ascending));
```

Comment:
It'll be a pain, but can we get nameof expressions for the sort descriptions for the parts that are at least our member names? Otherwise do we think we have enough tests that if we were to rename something and break this, we'll catch it?

Reply:
good call.